client/my-sites/plans-features-main/components/plan-type-selector/index.tsx

```diff
@@ -42,12 +35,14 @@ export type PlanTypeSelectorProps = {
 	selectedFeature?: string;
 	showBiennialToggle?: boolean;
 	isInSignup: boolean;
-	plans: string[];
+	plans: PlanSlug[];
```

Comment:
It's also possible to pass here `GridPlan[]` - albeit I have some concerns about keeping all the metadata under that structure i.e. we may end up splitting pricing from other GridPlan properties. Maybe something to come back to.